src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -1122,38 +1125,168 @@ class spell_pri_power_word_shield : public SpellScriptLoader
     }
 };
 
-// 33110 - Prayer of Mending Heal
-class spell_pri_prayer_of_mending_heal : public SpellScriptLoader
+// Base class used by various prayer of mending spells
+class spell_pri_prayer_of_mending_SpellScriptBase : public SpellScript
+{
+public:
+    bool Load() override
+    {
+        _spellInfoHeal = sSpellMgr->GetSpellInfo(SPELL_PRIEST_PRAYER_OF_MENDING_HEAL, DIFFICULTY_NONE);
```

Comment:
If you rely on existence of some spell, do the checking logic in Validate() to get startup logs